lib/block/operation_test.go

```diff
@@ -50,7 +50,7 @@ func TestBlockOperationSaveExisting(t *testing.T) {
 
 	bos := TestMakeNewBlockOperation(networkID, 1)
 	bo := bos[0]
-	bo.MustSave(st)
+	bo.Save(st)
```

Comment:
Yeah, those changes should also be reverted

Reply:
Sorry, I missed something. All done :)